source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -391,6 +392,7 @@ namespace AdaptiveSharedNamespace
         InvalidColorFormat,
         InvalidDimensionSpecified,
         InvalidLanguage,
+        CustomWarning,
```

Comment:
>CustomWarning [](start = 8, length = 13)

Similarly "CustomWarning" seems an odd enumeraiton value -- having a notion of Default seems more intuitive?